app/views/podcast_episodes/_episodes_feed.html.erb

```diff
@@ -17,7 +17,9 @@
         <h3><span class="tag-identifier">podcast</span><%= episode.title %></h3>
       </div>
       <h4>
-        <%= episode.podcast.title %>
+        <a href="/<%= episode.podcast.slug %>">
```

Comment:
I doubt it's ok to nest `a` tag inside an `a` tag. Though I agree that it's misleading that clicking on a podcast name leads to the episode page.

Reply:
https://github.com/araslanov-e/dev.to/blob/6a8c766322a9af31c97abc42370dcdf9854493b7/app/views/articles/index.html.erb#L119

Reply:
It's ok

Reply:
Well, the codebase doesn't only contain best practices. HTML doesn't allow nested links, but the browser does the job for us and pushes the nested link out of the parent one, you can check it in the source code.
For me, our solutions look like hacks but they could be acceptable, I'll aks other team members for their opinion.

Reply:
@lightalloy I think you're right in general... But in the longterm picture of cleaning this up I think it kind of makes sense to allow this change because then it makes future changes to the general practice more consistent as a whole.

I'm going to approve this but I'll leave it open for discussion if anyone disagrees.